src/EventStore.ClientAPI/Projections/ProjectionDetails.cs

```diff
@@ -0,0 +1,217 @@
+﻿using System;
+
+namespace EventStore.ClientAPI.Projections
+{
+    /// <summary>
+    /// Provides the details for a projection.
+    /// </summary>
+    public sealed class ProjectionDetails
+    {
+        /// <summary>
+        /// The CoreProcessingTime
+        /// </summary>
+        public readonly int CoreProcessingTime;
```

Comment:
Need to investigate whether this should be an `int` or some other type (perhaps `Timespan` given the name though I'm not sure what it actually represents without looking.